### ISSUE TITLE
Umožnit import FOXML dalších objektů

### ISSUE THREAD

**Issue by ZdenkaSera** [proarc/proarc#1726]
V současné době lze importovat FOXML exportované z Krameria pouze pro NDK jednosvazkové monografie a NDK periodika. Chtěli bychom umožnit import:
- Vícedílných monografií
- ...

**Comment by tumovam** [comment]
Nezapomneout na reimport k4 s eČlánky. Díky

**Comment by SykoraLukas** [comment]
Prosím o specifikaci jakých modelů se import týká + dodání odpovídajících foxml balíčků, které bude možné použít jako vzor.

**Comment by katerinasirova** [comment]
Do sdílené složky jsem nahrála monografii s kapitolami ze starší produkce (data neprošla zpracováním v PA ale v Siriovi, původně jsou v DTD, po konverzi je FOXML bez obrázků, ale s odkazy na imageserver - proto jsem do složky přidala i obrázky). Dejte prosím vědět, jestli je to takto v pořádku.

**Comment by SykoraLukas** [comment]
@katerinasirova jaké je uuid celé monografie? Prošel jsem všechny foxml a nenašel jsem titulovou úroveň. Kromě 2 kapitol jsem našel jen strany.

**Comment by SykoraLukas** [comment]
Dotaz na @tumovam a @katerinasirova 

Mají se strany rovnou překlápět na NDK strany? Nebo mají zůstat jako Strany?

**Comment by tumovam** [comment]
@SykoraLukas nyní se nám nedaří vyexportovat foxml z Krameria, balíček aktulizuji jakmile to půjde ;) Promiň za zdržení. 

A ty strany... já bych byla pro, tedy aby se Strany hned překlápěly na NDK Strany (máme z tomu i jiné rozvojové issue https://github.com/proarc/proarc/issues/1725 ).

**Comment by SykoraLukas** [comment]
Zatím vyřešeno:

- [ ] vícedílná monografie - pomocí nového profilu
- [ ] periodikum (resp. ročník, číslo) s NDK Články i eČlánky - importování pomocí profilu Kramerius NDK Periodikum. Vnitřní logika rozhodne, zda má jít o eČlánek s PDF nebo o NDK Článek se stranamu.

**Comment by tumovam** [comment]
> [@tumovam](https://github.com/tumovam) můžeš prosím zkontrolovat konzistenci baličku `d93ed756-1e79-44e8-acaa-8445733f7367_NDK Clanky` dle všeho tam chybí foxml pro `Číslo periodika` a tím pádem se to nedá složit.

Lukáši @SykoraLukas ve sdílené složce snad kompletní export vč. úrovně čísla - d93ed756-1e79-44e8-acaa-8445733f7367

**Comment by SykoraLukas** [comment]
Jak se chovat ke K4 Periodikum? Ma se to automaticky mitrovat na NDK modely?

**Comment by tumovam** [comment]
> Jak se chovat ke K4 Periodikum? Ma se to automaticky mitrovat na NDK modely?

@SykoraLukas ano. Pokud to je možné, ať se K4 při importu automaticky migrují na NDK, prosím :)

**Comment by SykoraLukas** [comment]
Zdá se, že pokud to necháme pod profilem Kramerius NDK Periodikum, tak to bude fungovat. Mohla bys prosím dodat nějaký vzorek K4 Periodika? Model sice dokážu pod profilem Kramerius NDK Periodikum převést na správný NDK model, ale potřebuji otestovat metadata a jejich transformaci. Zatím nemám vzorek dat, na kterém by se to dalo testovat, abych zjistil, jak bylo dříve popisováno.

**Comment by tumovam** [comment]
Jistě, asi to nebude hned, ale zkusíme něco najít. Díky

**Comment by tumovam** [comment]
> Mohla bys prosím dodat nějaký vzorek K4 Periodika? Model sice dokážu pod profilem Kramerius NDK Periodikum převést na správný NDK model, ale potřebuji otestovat metadata a jejich transformaci. Zatím nemám vzorek dat, na kterém by se to dalo testovat, abych zjistil, jak bylo dříve popisováno.

@SykoraLukas tak dvě K4 periodika jsou v ProAcr/Specifikace/Podklady k issues/API: 1726/Periodika_K4 - oba vzorky jsou trochu rozdílné mj. jedno ani nemá OCR. Příp. samozřejmě i k nahlédnutí v našem Krameriovi.

**Comment by SykoraLukas** [comment]
Díky, otestováno a zdá se, že to funguje. Případně poté lokální test a test na našem testovacím.

**Comment by SykoraLukas** [comment]
Je potřeba jen od @katerinasirova, aby zkontrolovala tu jejich složku a případně nahrála znovu. Tu jsem zatím nemohl vyřešit.

**Comment by tumovam** [comment]
> Zatím vyřešeno:
> 
>     * [x]  vícedílná monografie - pomocí nového profilu[x]  periodikum (resp. ročník, číslo) s NDK Články i eČlánky - importování pomocí profilu Kramerius NDK Periodikum. Vnitřní logika rozhodne, zda má jít o eČlánek s PDF nebo o NDK Článek se stranamu.

@SykoraLukas prosím jaký je nový profil pro reimport vícedílky, musíme lokálně upravit v konfiguraci? Díky

**Comment by tumovam** [comment]
Lokálně ve v2.4.2 (10.02.2026), jádro 4.3.2 otestováno foxml s eČlánky a NDK Články. Chybí otestovat VD a monografie s kapitolami (nemáme data).

**Comment by tumovam** [comment]
@SykoraLukas prosím jaký je nový importní profil pro reimport vícedílné monografie? Děkuji

**Comment by SykoraLukas** [comment]
Nový profil je `profile.ndk_monograph_title_kramerius_import`

**Comment by tumovam** [comment]
Lokálně jsme testovali reimport foxml VD pomocí profilu "Kramerius NDK Monografie s titulem", nejdříve se objeví:

<img width="1040" height="82" alt="Image" src="https://github.com/user-attachments/assets/bdb37c4e-414f-4bc0-9c18-e76098a31415" />

Plus ve Správě procesů:

<img width="1038" height="122" alt="Image" src="https://github.com/user-attachments/assets/c989f337-1dad-48e0-bfb1-b27f9ac2d2e6" />

NDK VD monografie + NDK svazky VD se naimportovaly správně, ale se Stranami, místo NDK Stran:

<img width="837" height="260" alt="Image" src="https://github.com/user-attachments/assets/78ad3faa-5c04-4979-8d47-5723670456df" />

**Comment by SykoraLukas** [comment]
Super, ty strany se řeší https://github.com/proarc/proarc/issues/1725 - mělo by to být v dalším releasu

**Comment by tumovam** [comment]
@katerinasirova mohli byste prosím vyzkoušet reimport FOXML monografie s kapitolami? My nemáme data, příp. je vyrobím, ale bylo by lepší to vyzkoušet na něčem starším. Děkuji

**Comment by katerinasirova** [comment]
Ano, čekáme na nasazení nové verze a vyzkoušíme.

**Comment by katerinasirova** [comment]
@SykoraLukas Podle dohody z minulé schůzky jsem nasdílela do sdílené složky nová data - stále platí, že foxml je bez obrázků, ty jsou zvlášť ve složce JP2 - máš to tam popsané v txt souboru -  ProAcr/Specifikace/Podklady k issues/API: 1726/1726-test_foxml_MKP_monografie_kapitoly/VB_14249.

**Comment by SykoraLukas** [comment]
Upraveno, aby to fungovalo i pro foxml s odkazy do Krameria musí mít importní balíček kromě jednotlivých xml souborů také jp2 soubory z imageserveru. Název souboru musí odpovídat poslední složce v cestě imageserveru. Poté import projde. Do složky, kam dala @katerinasirova jejich balíček ukládám ten upravený, jak musí být pro proarc.
